server/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -314,6 +314,13 @@ protected Node(final Environment initialEnvironment,
                     Build.CURRENT.getQualifiedVersion());
             }
 
+            if (initialEnvironment.dataFiles().length > 1) {
+                // NOTE: we use initialEnvironment here, but assertEquivalent below ensures the data paths do not change
+                deprecationLogger.deprecate(DeprecationCategory.SETTINGS, "multiple-data-paths",
+                    "Configuring multiple path.data paths is deprecated. Use RAID-0 or other system level features for utilizing" +
```

Comment:
Sorry, I should have thought of this when you ran this wording by me earlier. I think we should avoid "RAID0" here specifically, and simply say RAID, so that it doesn't seem like we're putting our weight behind a specific RAID configuration.